packages/block-library/src/html/preview.js

```diff
@@ -23,7 +23,7 @@ const DEFAULT_STYLES = `
 
 export default function HTMLEditPreview( { content, isSelected } ) {
 	const settingStyles = useSelect(
-		( select ) => select( blockEditorStore ).getSettings().styles
+		( select ) => select( blockEditorStore ).getSettings().styles || []
```

Comment:
I'm a bit unsure about this change, but it may cause styles to temporarily become undefined.

I needed to make this change to prevent the following error:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'filter') at preview.js:34:19
```

Reply:
You can also do something like `(settingStyles ?? [] ).filter( ( style ) => style.css )`.

P.S. That `useSelect` is missing the empty dependency array.

Reply:
Fixed by [7bcb7c0](https://github.com/WordPress/gutenberg/pull/66440/commits/7bcb7c059ec314ceca5ad8fa7ea6c3e935e8b7e7)